2024-03-20 04:44:32,598 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-20 04:44:32,598 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Configure stats pid to 75780
2024-03-20 04:44:32,598 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from C:\Users\CYP\.config\wandb\settings
2024-03-20 04:44:32,598 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from c:\Users\CYP\Documents\GitHub\AIAKI\wandb\settings
2024-03-20 04:44:32,598 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-20 04:44:32,599 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-20 04:44:32,599 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-20 04:44:32,599 INFO    MainThread:75780 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240320_044432-xkyj8j7j\logs\debug.log
2024-03-20 04:44:32,599 INFO    MainThread:75780 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240320_044432-xkyj8j7j\logs\debug-internal.log
2024-03-20 04:44:32,599 INFO    MainThread:75780 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001FC568B0580>
2024-03-20 04:44:32,599 INFO    MainThread:75780 [wandb_init.py:init():566] calling init triggers
2024-03-20 04:44:32,599 INFO    MainThread:75780 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'batch_size': 32, 'n_units': 8, 'learning_rate': 1e-05, 'lr_decay': 0.95, 'lr_step': 5, 'ns': 8, 'loss': 'mse'}
2024-03-20 04:44:32,599 INFO    MainThread:75780 [wandb_init.py:init():591] re-initializing run, found existing run on stack: kcsvn788
2024-03-20 04:44:32,600 INFO    MainThread:75780 [wandb_run.py:_finish():1970] finishing run dahs/IMV_LSTM_AKI_new/kcsvn788
2024-03-20 04:44:32,600 INFO    MainThread:75780 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-20 04:44:32,600 INFO    MainThread:75780 [wandb_run.py:_restore():2193] restore
2024-03-20 04:44:32,601 INFO    MainThread:75780 [wandb_run.py:_restore():2199] restore done
2024-03-20 04:44:38,219 INFO    MainThread:75780 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-20 04:44:38,221 INFO    MainThread:75780 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-20 04:44:38,229 INFO    MainThread:75780 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-03-20 04:44:38,349 INFO    MainThread:75780 [wandb_init.py:init():616] starting backend
2024-03-20 04:44:38,349 INFO    MainThread:75780 [wandb_init.py:init():620] setting up manager
2024-03-20 04:44:38,351 INFO    MainThread:75780 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-20 04:44:38,352 INFO    MainThread:75780 [wandb_init.py:init():628] backend started and connected
2024-03-20 04:44:38,354 INFO    MainThread:75780 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-20 04:44:38,354 INFO    MainThread:75780 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-20 04:44:38,354 INFO    MainThread:75780 [wandb_init.py:init():720] updated telemetry
2024-03-20 04:44:38,377 INFO    MainThread:75780 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-20 04:44:39,732 INFO    MainThread:75780 [wandb_run.py:_on_init():2262] communicating current version
2024-03-20 04:44:39,763 INFO    MainThread:75780 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.16.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-03-20 04:44:39,763 INFO    MainThread:75780 [wandb_init.py:init():804] starting run threads in backend
2024-03-20 04:44:39,884 INFO    MainThread:75780 [wandb_run.py:_console_start():2241] atexit reg
2024-03-20 04:44:39,884 INFO    MainThread:75780 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-20 04:44:39,884 INFO    MainThread:75780 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-20 04:44:39,884 INFO    MainThread:75780 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-20 04:44:39,884 INFO    MainThread:75780 [wandb_init.py:init():847] run started, returning control to user process
2024-03-20 06:29:54,262 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-20 06:29:54,263 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Configure stats pid to 75780
2024-03-20 06:29:54,263 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from C:\Users\CYP\.config\wandb\settings
2024-03-20 06:29:54,263 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from c:\Users\CYP\Documents\GitHub\AIAKI\wandb\settings
2024-03-20 06:29:54,263 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-20 06:29:54,263 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-20 06:29:54,263 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-20 06:29:54,263 INFO    MainThread:75780 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240320_062954-f15t1gs2\logs\debug.log
2024-03-20 06:29:54,264 INFO    MainThread:75780 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240320_062954-f15t1gs2\logs\debug-internal.log
2024-03-20 06:29:54,264 INFO    MainThread:75780 [wandb_init.py:init():566] calling init triggers
2024-03-20 06:29:54,264 INFO    MainThread:75780 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'batch_size': 32, 'n_units': 32, 'learning_rate': 1e-05, 'lr_decay': 0.95, 'lr_step': 10, 'ns': 8, 'loss': 'smooth_l1'}
2024-03-20 06:29:54,264 INFO    MainThread:75780 [wandb_init.py:init():591] re-initializing run, found existing run on stack: xkyj8j7j
2024-03-20 06:29:54,265 INFO    MainThread:75780 [wandb_run.py:_finish():1970] finishing run dahs/IMV_LSTM_AKI_new/xkyj8j7j
2024-03-20 06:29:54,265 ERROR   MainThread:75780 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-03-20 06:29:54,265 INFO    MainThread:75780 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-20 06:29:54,265 INFO    MainThread:75780 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-20 06:29:54,265 INFO    MainThread:75780 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-20 06:29:54,265 INFO    MainThread:75780 [wandb_run.py:_restore():2193] restore
2024-03-20 06:29:54,265 INFO    MainThread:75780 [wandb_run.py:_restore():2199] restore done
2024-03-20 06:29:59,501 INFO    MainThread:75780 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-20 06:29:59,503 INFO    MainThread:75780 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-20 06:29:59,509 INFO    MainThread:75780 [wandb_run.py:_footer_sync_info():3825] logging synced files
